torch/_dynamo/convert_frame.py

```diff
@@ -473,7 +473,10 @@ def _compile(
     output: Optional[OutputGraph] = None
     # This is shared across restarts
     mutated_closure_cell_contents: Set[str] = set()
+    fail_type: Optional[str] = None
     fail_reason: Optional[str] = None
+    fail_user_frame_filename: Optional[str] = None
+    fail_user_frame_lineno: Optional[int] = None
```

Comment:
TBH, I'm not so sure how useful the user frame is here; for framework problems, I want to aggregate on the TRUE exception raise site lol

Reply:
This is the innermost frame of user stack, which is the filename and lineno of where the error is from. It's the true exception raise site, right?

Reply:
There are two stacks for any Dynamo exception. The user stack, which represents where in the user code (e.g., model.py) that the problem arose, and the regular stack, which represents where in dynamo (e.g., torch/_dynamo/something.py) which got risen. It seems to me that you are reporting user stack here.

Reply:
Yea, I think we are going to report user stack, right? The regular stack info has been recorded at ```fail_reason```.

Reply:
Even if the regular stack was recorded in fail_reason in plaintext form, you would still want to record it in a column for easy aggregation. But I don't think it's recorded in fail_reason?

Reply:
Oh, I have two column for aggregation purpose, e.g.,
```
fail_type="<class 'torch._dynamo.exc.Unsupported'>", 
fail_reason='custome dict init with args/kwargs unimplemented', 
```
```fail_type``` is the type of exception, and ```fail_reason``` is the exception's plaintext message. Do you think this is good for aggregation?

Reply:
Or do you want to further extract some key words from the plaintext error message for aggregation? I don't think it helps a lot since they are unstructured data.

Reply:
So I guess, what I *really* want, is a stable identifier for every error site which I can use to aggregate and see when I am hitting error sites. A proxy for this is the filename:line of the error line, although this will change over time. Maybe we should just start allocating error codes for every error in Dynamo.

Reply:
Yes, I think we should allocate error code for every error in Dynamo. I don't like the idea of using filename:line since it's always changing.